aclk/README.md

```diff
@@ -1,10 +1,105 @@
 <!--
 ---
 title: "Agent-cloud link (ACLK)"
+description: "The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. 
+The ACLK uses MQTT over secure websockets to create, persist, and encrypt the connection from end-to-end, and enable 
+the features found in Netdata Cloud."
+date: 2020-03-23
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connection from
+end-to-end, and enable the features found in Netdata Cloud.
+
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
+
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
+your agent to Netdata Cloud via the ACLK. Data used to render charts and alarms in Netdata Cloud is accessed directly
+from agents through the ACLK. Though the data does flow through Netdata servers on its way from agents to the browser,
+it is never logged or stored.
+
+Read our [claiming documentation](../claim/README.md) for details on node claiming.
+
+## Enable and configure the ACLK
+
+The ACLK is enabled by default if its prerequisites installed correctly, and the correct configuration is already set in
+your agent's `netdata.conf` file.
+
+```conf
+[cloud]
+    cloud base url = https://up.netdata.cloud
+```
+
+### Proxy configuration
+
+If your Netdata agent uses a proxy to reach the outside internet, you must configure a SOCKS5 proxy in the
+`[agent_cloud_link]` section of your `netdata.conf` file. By default, the section looks like the following:
+
+```conf
+[agent_cloud_link]
+    proxy = env
+```
+
+The `proxy` setting takes a few different values:
+
+-   `env`: Netdata reads the environment variables `http_proxy` and `socks_proxy` to discover the correct
```

Comment:
maybe we can add that if both are defined `socks_proxy` is preferred. If this is not satisfactory and it is not desired to modify environment variables setting fixed proxy will work (as per info 2 lines below).

Reply:
Updated this section with some more information. Let me know if it looks OK.